### ISSUE TITLE
Default zoom in map header template not working.

### ISSUE THREAD

**Issue by KacperSzyf** [archesproject/arches#8218]
**Describe the bug**

When adding a map header to a report, the default zoom option does not do anything. 
The map on the header is always zoomed all the way in.

**To Reproduce**
1. Go to 'Arches Desinger > Report > Cards > Header'
2. Click on 'Template'
3. Scroll down to 'Map Header Template'
4. Change 'Default Zoom' 

**Expected behavior**
Zoom level chaning in the report view when default zoom is changed.

**Your Arches Information**

*   Version used:6
*   Operating System and version (desktop or mobile):Win11 21H2
*   Browser Name and version: Chrome 99

**Comment by yshng** [comment]
@chiatt We are having the same problem, and I believe I understand from your responses why the configuration that exists right now does not help resolve the default zoom once the actual features are loaded. I understand your point about setting max zoom globally based on the detail of the basemap, however, the issue is not as much about basemap resolution so much as UX. When initially presenting a resource on the map, we want to show more geographic context, but still allow the user to zoom closer for more detail if desired (as the map can support a higher zoom level, but that higher zoom level is not our desired initial presentation). Can this functionality be added?

**Comment by CWDamm-Kint** [comment]
Adding to this issue, as we have had the same point raised by a client recently. As outlined by @yshng, the limitation of using maximum zoom in system settings is that users might want the initial view in the report to be more zoomed out,  but to still have the option of zooming in further manually.

It feels as though there may be a need for the report to have it's own default zoom setting. Or perhaps an 'offset' setting, so that the zoom still defaults to the bounding box of the geometry (or max zoom for single points), but you can get it to automatically zoom out a set amount from there.

**Comment by yshng** [comment]
@CWDamm-Kint Take a look [here](https://community.archesproject.org/t/adjust-default-zoom-extent-for-point-geometry-on-reports/542/9). I was able to use this approach to set a different default zoom for report headers, [like so](https://code.librehq.com/ots/arches/arches-projects/-/blob/vmhr-main/arches_project/media/js/utils/map-configurator.js?ref_type=heads).

**Comment by CWDamm-Kint** [comment]
Thanks @yshng, that's really helpful. 

Given that a lot of people seem to have asked the same question over the years, I wonder whether it might still be worth having something explicit in the system settings for admin users to adjust, but this is definitely a good workaround for developers.

**Comment by yshng** [comment]
Yes, @CWDamm-Kint, definitely agree that this workaround doesn’t take away from the desire to have a built-in UI setting that non-dev admins can use, only hope that it might help you in this moment!
